flybywiresim/a32nx
# Hoppie fixes

Fixes several reports in the discord about callsign in use and silent errors with TELEX

## Summary of Changes
- Fixes the callsign in use reports due to multiple parallel logins
- Fixes behaviour in the logon code validation
- Fixes ignored callsigns if TELEX is disabled

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. fix
- deactivate TELEX
- deactivate Hoppie
- Enter C/S in INIT page

2. fix
- EFB/Settings/ ATSU/AOC
- enter invalid Hoppie code -> See Hoppie error
- enter valid Hoppie code -> No error triggered


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
